temporal/api/cloud/nexus/v1/message.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
+
+package temporal.api.cloud.nexus.v1;
+
+option go_package = "go.temporal.io/api/cloud/nexus/v1;nexus";
+
+import "google/protobuf/timestamp.proto";
+
+message EndpointSpec {
+    // The name of the endpoint. Must be unique within an account.
+    // The name must match `[a-zA-Z_][a-zA-Z0-9_]*`.
+    // This field is mutable.
+    string name = 1;
+
+    // Indicates where the endpoint should forward received nexus requests to. 
+    EndpointTargetSpec target_spec = 2;
+
+    // The set of authorization policies for the endpoint. Each request's caller
+    // must match with at least one of the specs to be accepted by the endpoint.
+    // This field is mutable.
+    repeated EndpointAuthorizationSpec authorization_specs = 3;
+
+    // Optional description. Can be in markdown for rendering in the UI
+    string description = 4;
```

Comment:
delta from OSS: that is a temporal api payload.
we have a string here and user would need to take care of encryption/sensitive data out of band (or link to a website which does auth)

Reply:
I think this should be payload to be future proof.

Reply:
for now, add comment: experimental, this data type may change in the future

Reply:
added comment